### ISSUE TITLE
Need more pages?

### ISSUE THREAD

**Issue by dreadnaut** [Utodev/TurboPascalHelp#1]
Hello there!
I stumbled on this repository, and noticed that you tried to mirror my partial translation of the Turbo Pascal help. I've never completed the project—there were so many pages!—but I do have a full HTML dump of the English version if you need.

I could also publish my old Pascal code to extract pages, it might still work :)

**Comment by Utodev** [comment]
Hello!

Yes, I was creating new software with Turbo Pascal some months ago, and during that timeI found your Turbo Pascal help, and it was very useful, as I didn't have to open the Turbo Pascal IDE. I'm not Italian but Spanish, but it seems my memories from the 90s and  what I can understand in Italian were enough ;)

To be honest, I found your page was the only place where Turbo Pascal Help was stored only, so my only intention when cloning the information was to have a backup.  In fact, during those days I found many DOS information is already difficult to find in the www, so your site was not the only thing I saved.

Regarding the Turbo pascal help in English and HTML, it would be definitively helpful, and IMO it would be nice to have it in a repository, but otherwise I can host it in my server too.

Thanks for contacting, and thanks for your work, wheter finished or not, it was very helpful.

**Comment by Utodev** [comment]
Hi! Did you finally manage to find the English dump of TP Help?

Thanks!

**Comment by dreadnaut** [comment]
Yes, I found all the pieces including the generator. I'll try to update it to generate better HTML, but for the moment the dump is here: https://dreadnaut.altervista.org/pascal/tp7eng.zip

**Comment by Utodev** [comment]
Sorry for the delay in response, I somehow missed your mail. Thank you very
much, I'll have a look!

**Comment by Utodev** [comment]
Hi there, today I was looking for the english TPH extracted files and came back to this issue when googling for them. Also I found someone asking for our TPH2HTML extractor which is apparently not available in our website. Maybe it never was there.

I don't know how good it worked, but I think it would be great to have it and its source code online somehow, as this kind of stuff it's starting to be difficult to find in the Internet.

I will be creating anothe repository with the english files you provided me, giving you full credit, just so it's avaiable in Github, where they be useful for other people. I hope you don't mind, but if so, just let me know and I will remove it.

If you can get the extractor, whether the executable, the sources, or both, they can still be helpfult to extract othere TPH files (i.e. TVISION.TPH).

Thanks again for your work!

**Comment by dreadnaut** [comment]
I'm actually working on a web page for my Turbo Pascal code, so it should be live soon! The TPH contents are not mine, so I have no voice on that 😀
